stopes/ui/seamlisten/backend/app/fileviewer.py

```diff
@@ -86,6 +85,7 @@ def open_segment_tsv(file: str, start_idx: int, end_idx: int) -> tp.List[LineRes
     with stutils.open(file) as f:
         for line in itertools.islice(f, start_idx, end_idx):
             results.append(auto_parse_line(line.strip()))
+            print("result", results)
```

Comment:
please remove all the debug print statements.

Reply:
done